heartbeat/ocf-shellfuncs.in

```diff
@@ -101,7 +101,7 @@ ocf_is_decimal() {
 
 ocf_is_true() {
 	case "$1" in
-	yes|true|1|YES|TRUE|ja|on|ON) true ;;
+	yes|true|1|YES|TRUE|True|ja|on|ON) true ;;
```

Comment:
It would be easier to use

    case "$(echo "$1" | tr '[:upper:]' '[:lower:]')" in

Reply:
Strictly as a point of comparison, pacemaker's crm_is_true() function is similar except it used a case-insensitive match, adds 'y', and omits 'ja'.

Reply:
Yeah, but I guess this covers all use-cases except if someone uses caps-lock and shift :)

Reply:
Wouldn't it make sense to align this with the pacemaker function? At least enabling `y` makes a lot more sense than `ja`, and I doubt using `ja` would work anyway :D

Reply:
It wouldn't hurt, but I don't think it's necessary. Pacemaker uses its function for pacemaker cluster properties and resource meta-attributes. Resource agents mostly use it for their own resource parameters, so there's no chance of problems there. If they use it on pacemaker meta-attributes, there could be, but I don't think they would check the few boolean properties available now (I believe only is-managed and allow-migrate).

Reply:
I suppose the main argument for it would simply be to avoid confusion for users about what is allowed. We've definitely seen that in time values, where pacemaker accepts ms/secs/etc. units while some resource parameters don't accept units.

Reply:
Yeah, if nothing else it's inconsistent.. I guess the best thing would be if the OCF standard defined which values are truthy, and both pacemaker and the agents conformed to that.